File: JS/backend/entities/empleado.js
class Empleado {
    constructor(nombre, apellido, telefono, mail, rol) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.telefono = telefono;
        this.mail = mail;
        this.rol = rol;
    }
    obtenerMail() {     // método de ejemplo
        return this.mail;
    }
}

File: Python/entidades/empleado.py
from .rol import Rol

class Empleado():
    def __init__(self, nombre, apellido, mail, telefono, rol: Rol):
        self.nombre = nombre
        self.apellido = apellido
        self.mail = mail
        self.telefono = telefono
        self.rol = rol

    def getMail(self):
        return self.mail
    
    def esResponsableReparacion(self):
        return self.rol.esResponsableReparacion()